MobileFilter/sections/specific_web.txt

```diff
@@ -44,6 +44,7 @@ trinixy.ru##div.mdl-card[style^="text-align: center;"]
 jukenbbs.com##div[style="margin-top:1px; margin-bottom:1px;"]
 jukenbbs.com##div[style="margin-top:10px; margin-bottom:10px; text-align:center;"]
 sirogohan.com##.sp-banner
+full-count.jp##.erase_space-ad
```

Comment:
Not much want to hide entire this for the same reason as https://github.com/AdguardTeam/AdguardFilters/commit/cc85b4d94882cc02a9a7fce4580730cdd5af15c5#r70606842

Reply:
To Annoyances?